reports/sdks.go

```diff
@@ -5,59 +5,51 @@ import (
 	"fmt"
 	"io"
 	"net/http"
-	"os"
 	"regexp"
 	"strings"
 
-	"github.com/google/go-github/v57/github"
 	"golang.org/x/exp/slog"
 )
 
 var (
-	ghToken = os.Getenv("GITHUB_TOKEN")
-	gh      = github.NewClient(nil).WithAuthToken(ghToken)
-	SDKs    = []SDKMeta{
+	SDKs = []SDKMeta{
 		{
 			Name:         "web5-js",
 			Repo:         "TBD54566975/web5-js",
 			ArtifactName: "junit-results",
 			FeatureRegex: regexp.MustCompile(`Web5TestVectors(\w+)`),
 			VectorRegex:  regexp.MustCompile(`\w+ \w+ (\w+)`),
+			VectorPath:   "test-vectors",
 		},
 		{
 			Name:         "web5-kt",
 			Repo:         "TBD54566975/web5-kt",
 			ArtifactName: "test-results",
 			FeatureRegex: regexp.MustCompile(`web5\.sdk\.\w+.Web5TestVectors(\w+)`),
 			VectorRegex:  regexp.MustCompile(`(\w+)\(\)`),
+			VectorPath:   "test-vectors",
 		},
 	}
 )
 
-func init() {
-	if ghToken == "" {
-		panic("please set environment variable GITHUB_TOKEN to a valid github token (generate one at https://github.com/settings/tokens?type=beta)")
-	}
-}
-
 func GetAllReports() ([]Report, error) {
 	ctx := context.TODO()
 
 	reports := []Report{}
```

Comment:
nit: `var reports []Report`

Reply:
Out of scope of this PR, but it doesn't seem to break anything so I'll roll it in. Not sure why I thought I had to initialize slices before I could append to them